backgrid-paginator.js

```diff
@@ -442,7 +442,21 @@
       this.el.appendChild(ul);
 
       return this;
-    }
+    },
+	remove: function () {
+		if (this.handles) {
+			for (var i = 0, l = this.handles.length; i < l; i++) {
+				this.handles[i].remove();
+			}
+		}
+		// COMPLETELY UNBIND THE VIEW
+		this.undelegateEvents();
```

Comment:
You can just call line 458 and these 3 lines should have been done for you already right?